include/common_package/common_node.hpp

```diff
@@ -12,14 +12,45 @@
 #include "rclcpp/rclcpp.hpp"
 #include "topic_names.hpp"
 
-
 // Message includes
 #include "interfaces/msg/heartbeat.hpp"
 #include "interfaces/msg/job_finished.hpp"
 
 using namespace std::chrono_literals;
 
 namespace common_lib {
+
+namespace {
+
+/**
+ * @brief rosout QoS to explicitly set the it
```

Comment:
What do you want to say here?

Reply:
```suggestion
 * @brief rosout QoS to explicitly set the rosout for  Nodes subclassing CommonNode
```

Better?

Reply:
Yes, now it makes sense for me

Reply:
Autoformatter doesn't like this chance 😂
@J0las

Reply:
Mimimimi